optimism.tokenlist.json

```diff
@@ -659,6 +663,28 @@
       "extensions": {
         "optimismBridgeAddress": "0x0CFb46528a7002a7D8877a5F7a69b9AaF1A9058e"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x01ba67aac7f75f647d94220cc98fb30fcc5105bf",
+      "name": "Lyra",
+      "symbol": "LYRA",
+      "decimals": 18,
+      "logoURI": "https://www.lyra.finance/images/logo.png",
```

Comment:
Add an image for Lyra to images folder and link to it here. Otherwise the token will not display properly in the gateway dropdown.